app/steps/ui/starteligibility/template.html

```diff
@@ -0,0 +1,29 @@
+{% extends "includes/layout.html" %}
+{% block page_title %}
+{{ content.title }}
+{% endblock %}
+
+{% block content %}
+
+{% from "widgets/lists.html" import taskAction, unorderedList %}
+
+<main id="content" role="main">
+    {% include "includes/phase_banner_beta.html" %}
+    <div class="grid-row">
+        <div class="column-two-thirds">
+            <h1 class="form-title heading-large">{{content.header}}</h1>
+            <p>{{content.introduction | safe}}</p>
+            <p>{{content.paragraph1 | safe}}</p>
+            <p>{{content.paragraph2 | safe}}</p>
```

Comment:
Same as before, this may not require 'safe'

Reply:
This content contains an html link so will need to be `safe`

Reply:
paragraph 2 didn't have a link, it is:
"paragraph2": "We will invite you to try this new service if you can confirm that you have:",